templates/zerver/api/writing-bots.md

```diff
@@ -30,14 +30,15 @@ On this page you'll find:
 
 2. `cd python-zulip-api` - navigate into your cloned repository.
 
-3. `./tools/provision` - install all requirements in a Python virtualenv.
+3. `python3 ./tools/provision` - install all requirements in a Python virtualenv.
 
-4. Run the `source <activation/path>` command printed in the previous
-   step to activate the virtualenv.
+4. `source /.../python-zulip-api/.../activate` - run the `source` command printed
+   at the end when `python3 ./tools/provision` finishes.
```

Comment:
I think we can write this more clearly.  "The output of `provision` will end with a command of the form `source .../activate`; run that command to enter the new virtualenv.